src/patternfly/layouts/Stack/stack-item.hbs

```diff
@@ -1,3 +1,3 @@
-<div class="pf-l-stack__item{{#if modifierClass}} {{ modifierClass }}{{/if}}">
+<div class="pf-l-stack__item {{#if stack-item--modifier}}{{stack-item--modifier}}{{/if}}">
```

Comment:
Remove the space between `pf-l-stack__item {{#if` and add a space between `{{#if stack-item--modifier}}{{stack-item--modifier}}`

Reply:
done